pkg/components/references.go

```diff
@@ -78,6 +78,11 @@ func GetReference(c Component, registry, imagePath, imagePrefix string, is *oper
 		if c.Registry != "" {
 			registry = c.Registry
 		}
+	} else if !strings.HasSuffix(registry, "/") {
+		// If the registry is explicitly set, make sure it ends with a slash so that the
+		// image can be appended correctly below.
+		// Make sure registry, except for the special case "UseDefault", always ends with a slash.
```

Comment:
I think this last comment line is superfluous and the 1st 2 lines of comments are sufficient.